src/core/server/deprecations/routes/get.ts

```diff
@@ -15,7 +15,7 @@ export const registerGetRoute = (router: IRouter) => {
       validate: false,
     },
     async (context, req, res) => {
-      const deprecationsClient = context.core.deprecations.client;
+      const deprecationsClient = (await context.core).deprecations.client;
```

Comment:
There's a subtle difference in the way we're accessing `deprecations` response vs accessing the [`defaultRoute`](https://github.com/elastic/kibana/pull/129896/files?file-filters%5B%5D=.js&file-filters%5B%5D=.md&file-filters%5B%5D=.ts&owned-by%5B%5D=TinaHeiligers&show-viewed-files=true#diff-c74b7c2b8f2f0a9eb1c208f641c0dde8f77a101f70909b72deea60d38c9003e2R92-R93) that I'm missing somewhere. Not sure where though...